<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Common_Inputs</class>
 <widget class="QWidget" name="Common_Inputs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="IgnWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QRadioButton" name="IGN_ACC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IGN_ACC</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="IGN_OFF">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IGN_OFF</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="IGN_ON">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IGN_ON</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="IGN_CRANK">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IGN_CRANK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
